packages/app/src/app/pages/Live/index.js

```diff
@@ -1,105 +1,111 @@
-import * as React from 'react';
-import { inject, observer } from 'mobx-react';
-import { Link } from 'react-router-dom';
-
-import { getSandboxName } from '@codesandbox/common/lib/utils/get-sandbox-name';
-import Navigation from 'app/pages/common/Navigation';
+import Centered from '@codesandbox/common/lib/components/flex/Centered';
 import Fullscreen from '@codesandbox/common/lib/components/flex/Fullscreen';
+import Padding from '@codesandbox/common/lib/components/spacing/Padding';
+import { getSandboxName } from '@codesandbox/common/lib/utils/get-sandbox-name';
+import { observer } from 'mobx-react-lite';
```

Comment:
I prefer to keep the order:

- Libraries (React first)
- Our libraries (@codesandbox/common)
- Absolute paths to project files
- Relative paths

Does that make sense?

Reply:
I added the own libraries at the  top, just because technically it's a different package that could come from elsewhere, but if you want I can move them below 🙂